DataFormats/HcalCalibObjects/interface/HOCalibVariables.h

```diff
@@ -1,13 +1,13 @@
 #ifndef AlCaHOCalibProducer_HOCalibVariables_h
 #define AlCaHOCalibProducer_HOCalibVariables_h
 #include <vector>
-//April 2015 : Added more variables, e.g., momatho, tkpt03, ecal03, hcal03, inslumi, nprim
+//April 2015 : Added more variables, e.g., momatho, tkpt03, ecal03, hcal03, pileup, nprim
 class HOCalibVariables {
 public:
   int nmuon;  //number of muons in the event
   int nprim;  // number of primary vertices
 
-  float inslumi;  //instantaneous luminosity
+  float pileup;  //Number of pileup events in the bunch crossing
```

Comment:
@cms-sw/alca-l2 

Maybe a naive question: does such renaming warrant a "ioread" rule ([example](https://github.com/cms-sw/cmssw/blob/a52f2d1ecf97bde0e9f10d45959c88337c481687/DataFormats/CTPPSReco/src/classes_def.xml#L31-L33)), to be able to read existing files that contain `ClassVersion="16"` of this product (if any) even after the data format is updated to `ClassVersion="17"` ?

Reply:
hi @missirol I think you are right, certainly it would be nice if the V16 files could be read with V17. @bsunanda can you quickly follow-up on this?